flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -952,7 +952,7 @@ private boolean transitionState(ExecutionState currentState, ExecutionState newS
 			if (cause == null) {
 				LOG.info("{} ({}) switched from {} to {}.", taskNameWithSubtask, executionId, currentState, newState);
 			} else {
-				LOG.info("{} ({}) switched from {} to {}.", taskNameWithSubtask, executionId, currentState, newState, cause);
+				LOG.warn("{} ({}) switched from {} to {}.", taskNameWithSubtask, executionId, currentState, newState, cause);
```

Comment:
This will not work because the number of placeholders (`{}`) does not match the number of arguments.

Reply:
Nevermind, you didn't change that.

Reply:
the last argument is a throwable, so I don't see a problem.

Reply:
Yes, I realized that and wrote my second comment.